igvc_ws/src/igvc_ekf/include/igvc_ekf/ekf.h

```diff
@@ -15,8 +15,8 @@
 #define NSEC_TO_SEC (double)(1000.0 * 1000.0 * 1000.0)
 
 // Robot Properties
-#define WHEELBASE_LEN (double)(0.84455) // 33.25 inches (in meters)
-#define WHEEL_RADIUS (double)(0.127)    // 5 inches (in meters)
+#define WHEELBASE_LEN (double)(0.66) // meters
+#define WHEEL_RADIUS (double)(0.104) // meters
```

Comment:
I'm not sure why these values were changed; are they more correct this way? In either case, the ekf.h file is not used, and the robot characteristics that get used by the EKF are defined near the top of the ekf_dead_reckoning.py file.

Reply:
These were for the 8in wheels, forgot to change them back. I was waiting to put everything into master first so the EKF message actually exists in the same branch before changing things over. Thanks for catching

Reply:
Ah gotcha. I merged master into this branch so that we could go ahead and get it working with the new EKF before merging.